src/com/owncloud/android/lib/resources/files/ReadRemoteFolderOperation.java

```diff
@@ -171,6 +171,7 @@ private RemoteFile fillOCFile(WebdavEntry we) {
         file.setSize(we.size());
         file.setQuotaUsedBytes(we.quotaUsedBytes());
         file.setQuotaAvailableBytes(we.quotaAvailableBytes());
+        file.setPrivateLink(we.privateLink());
```

Comment:
Could you please rename this variable **we** as **webdavEntry**? Clean code philosophy, you know...

Reply:
Done